app/src/main/java/com/xmartlabs/taskloans/data/model/DataEntry.kt

```diff
@@ -0,0 +1,12 @@
+package com.xmartlabs.taskloans.data.model
+
+import com.xmartlabs.taskloans.data.model.service.UserResponse
+import java.util.Date
+
+data class DataEntry(
```

Comment:
If you are using it only fore service response I'd rename to DataEntryServiceResponse

Reply:
Or if it's used in other places I'd call it `Entry` or `TaskEntry`